beast/observationmodel/observations.py

```diff
@@ -125,35 +99,33 @@ def setFilters(self, filters):
 
     def getMags(self, num, filters):
         raise Exception('Do not use as magnitudes')
-        return np.array([ self.data[tt][num] - self.distanceModulus for tt in filters])
```

Comment:
Another way of saying this is that the distance should be used in the physicsmodel code, but not in the observationsmodel code.